### ISSUE TITLE
AdminUserModule not part of exported nestjs-admin modules

### ISSUE THREAD

**Issue by fechy** [Theodo-UK/nestjs-admin#165]
Trying to create a custom admin module as described in ./docs/custom-admin-module.md is not possible since AdminUserModule cannot be imported from 'nestjs-admin':
```
app/node_modules/nestjs-admin/dist/src"' has no exported member 'AdminUserModule'
```
<img width="633" alt="Screenshot 2020-05-07 at 10 28 37" src="https://user-images.githubusercontent.com/1142092/81272549-e8c00000-904d-11ea-8991-95a9bd2bf350.png">

I'm using the latest version of both nestjs (7.0.9) and nestjs-admin (0.4.0).

**Comment by anonrig** [comment]
I'm having the same exact issue, did you solve this? @fechy

**Comment by fechy** [comment]
@anonrig unfortunately not, nor I have the time right now to fix it. I'll try to find a way in a few days if the author doesn't come back with a solution himself.

**Comment by TheBigBadBunce** [comment]
For anyone else who stumbles across this, it's fixed in #188. Updating to version 0.4.1 should fix the problem!
